netlify.toml

```diff
@@ -3,7 +3,7 @@ publish = "build/"
 command = "CI='' npm run build"
 
 [build.environment]
-NODE_VERSION = "18"
+NODE_VERSION = "20.5.0"
```

Comment:
<img width="472" alt="image" src="https://github.com/coronasafe/care_fe/assets/29787772/cb57a24b-b33a-4ee6-a41e-61a85c88d310">

can we switch this to one of the LTS version